src/clang_parser.cpp

```diff
@@ -886,4 +890,33 @@ CXUnsavedFile ClangParser::get_empty_btf_generated_header()
   };
 }
 
+std::string ClangParser::get_arch_include_path()
+{
+  struct utsname utsname;
+  uname(&utsname);
+  LOG(INFO) << utsname.machine;
+  return "/usr/include/" + std::string(utsname.machine) + "-linux-gnu";
+}
+
+std::vector<std::string> ClangParser::system_include_paths()
+{
+  auto clang = "clang-" + std::to_string(LLVM_VERSION_MAJOR);
+  auto cmd = clang + " -Wp,-v -x c -fsyntax-only /dev/null 2>&1";
+  auto check = exec_system(cmd.c_str());
```

Comment:
This will throw if `clang` does not exist, right? We need to catch `std::runtime_error` here and handle it

Reply:
It won't, `exec_system` calls `popen` which doesn't fail on non-existing command. Anyway, it may fail in other situations, so I added exception handling.